scripts/update/Updater.php

```diff
@@ -36,7 +36,7 @@ class Updater extends common_ext_ExtensionUpdater
      */
     public function update($initialVersion)
     {
-        $this->skip('0.0.1', '0.9.0');
+        $this->skip('0.0.1', '0.10.0');
```

Comment:
You have to update the role in the updater as well

Reply:
This extension is not used on prod yet. So it's not necessary

Reply:
yep, I see, but this is possible that developers will have an issue with roles in the future

Reply:
Done. Please review it again.